### DISCUSSION TITLE
Access to Link Issue

### DISCUSSION THREAD

**Discussion by lhammerbh** [tabulartools/pbi-explorer#19] [Q&A]
Good PBI Explorer people

When trying to download, I get this error message. We are using DarkMail Trace, and somehow the link is blocked.
![image](https://github.com/tabulartools/pbi-explorer/assets/69352441/a0232fed-347a-471a-87b3-55162552648b)
There is probably nothing you can do about it, but I thought you should know that there can be issues with the link, when companies are using things like DarkMail Trace.

Yours
Lars Hammer

**Comment by DBojsen** [comment]
Hi Lars

That's a shame - can your IT department help enable the download?

**Comment by lhammerbh** [comment]
Thanks David

I am able to download now.


Best regards,
Bunker Holding A/S

***@***.***
Lars Christian Hvidegaard Hammer
Technical Business Intelligence Lead
Group IT

M
+45 60 37 67 98
***@***.***<mailto:(Email)>

VAT number: 75266316
[Bunker Holding Group logo]
Terms, conditions, and privacy policy of the company can be found at www.bunker-holding.com<http://www.bunker-holding.com>
